WiinUPro/Windows/MainWindow.xaml.cs

```diff
@@ -25,7 +25,7 @@ public static object CurrentTab
             }
         }
 
-        private static MainWindow _instance;
+        public static MainWindow _instance;

```

Comment:
Let's keep this private, and I'll add a comment where relevant on an alternative way other than reaching into this class directly.